KarhooSDK/Api/DataStore/User/Credentials.swift

```diff
@@ -8,21 +8,37 @@
 
 import Foundation
 
-struct Credentials {
+public struct Credentials {
     let accessToken: String
     let expiryDate: Date?
     let refreshToken: String?
-    
-    init(accessToken: String, expiryDate: Date?, refreshToken: String?) {
+    let refreshTokenExpiryDate: Date?
+
+    init(
```

Comment:
needs to be public in order for the DP to instantiate the Credentials we want them to pass on to us.